test/ng/directive/formSpec.js

```diff
@@ -386,6 +386,9 @@ describe('form', function() {
       doc = jqLite('<form ng-submit="submitMe()">' +
                      '<input type="submit" value="submit">' +
                    '</form>');
+      // Support: Chrome 60+ (on Windows)
+      // We need to add the form to the DOM in order for `submit` events to be properly fired.
+      window.document.body.appendChild(doc[0]);
```

Comment:
Aren't all of these memory leaks? Or is there some global cleanup between tests?